scrapy/utils/project.py

```diff
@@ -71,3 +71,16 @@ def get_project_settings():
         settings.setdict(env_overrides, priority='project')
 
     return settings
+
+def get_project_path():
+    """Return the Python path of the current project"""
+    # XXX: This is far from optimal. It fails when the settings module lives
+    #      outside of the project package, and when the project package lives
+    #      within another package.
+    if not inside_project():
+        raise NotConfigured("Not inside a project")
+    settings_module_path = os.environ.get(ENVVAR)
+    if not settings_module_path:
+        raise NotConfigured("Unable to locate project's python path")
+    return settings_module_path.split('.')[0]
```

Comment:
I think we could add another section like the `[settings]` one in scrapy.cfg to point out the addons folder. A hacky workaround to provide backward compatibility if this new section is not present could be to take the the project path directly from `[settings]` I think.